flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/BackPressureSampleableTask.java

```diff
@@ -22,13 +22,13 @@
 import org.apache.flink.runtime.executiongraph.ExecutionAttemptID;
 
 /**
- * Task interface used by {@link StackTraceSampleService} for back pressure tracking.
+ * Task interface used by {@link BackPressureSampleService} for back pressure tracking.
  */
-interface StackTraceSampleableTask {
+public interface BackPressureSampleableTask {
 
 	boolean isRunning();
 
-	StackTraceElement[] getStackTrace();
+	boolean isBackPressured();
 
 	ExecutionAttemptID getExecutionId();
```

Comment:
This interface method is also not very necessary, and the only usage atm is for throwing exception. I guess we can remove it as well.